src/collisionAvoidence.cpp

```diff
@@ -26,7 +28,8 @@ void SubAndPub::cmdVelCallback(const sensor_msgs::LaserScan::ConstPtr& scan_msg)
   bool shouldTurn = false;
   double sumOfLeftSide = 0.0;
   double sumOfRightSide = 0.0;
-  for (int i = scan_msg->ranges.size() / 3; i < (scan_msg->ranges.size() - (scan_msg->ranges.size() / 3)); ++i) {
+  if(LIDAR_IS_ON_THE_CORRECT_WAY){
```

Comment:
I'm not a huge fan of this patch. By making this boolean `const` the compiler will have an attitude of _Oh, ok. He is a silly programmer who wrote an if statement about a thing that doesn't chagnge. Let me just remove it._

Reply:
Ahhh yes that makes a lot of sense, changing a ROS param would be like a zillion times more user friendly, and using tf2 like that would be even better. It's so cool to me how every level of ROS can really work together! :sunglasses: I'll try to blast that patch either tn or early tmrw.

Reply:
take your time with it! And no worries -- it's hard to get used to this mindset (don't forget to pull before you start patching, as I did push the linter changes to this branch).